website_multi_theme/models/website.py

```diff
@@ -36,6 +36,10 @@ class Website(models.Model):
         help="Views generated by the multiwebsite theme just for this website",
     )
 
+    base_url = fields.Char(
+        string='Base Url'
```

Comment:
No need to specify the string. Simply writing `base_url = fields.Char()` will behave exactly the same.